pandas_test.py

```diff
@@ -43,25 +46,28 @@ def data_frame_test():
     #df[['Close', 'Adj Close']].plot()
 
     # Column indexing
-    print df['Close'][:10]  # (and row indexing)
-    print df['Close'].max()
-    print df['Close'].mean()
-    print df['Close'].std()
+    print(df['Close'][:10])  # (and row indexing)
+    print(df['Close'].max())
+    print(df['Close'].mean())
+    print(df['Close'].std())
+    print(df[['Open', 'Close']].mean())  # multi column
 
     # Row indexing
-    print df.iloc[0]  # Integer based
-    print df.loc['2017-02-02']  # Label based
+    print(df.iloc[0])  # Integer based
+    print(df.iloc[0:3])  # Integer based, multi row
+    print(df.loc['2017-02-06'])  # Label based
+    print(df.loc['2017-02-06':'2017-02-10'])  # Label based, multi row
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Slice index is not an int, None, or instance with __index__](https://app.codacy.com/manual/mathieugouin/tradesim/pullRequest?prid=5336609)